sympy/functions/elementary/exponential.py

```diff
@@ -198,7 +199,14 @@ def as_base_exp(self):
         return ExpBase.as_base_exp(self)
 
 
-class exp(ExpBase):
+class ExpMeta(FunctionClass):
+    def __instancecheck__(cls, instance):
+        if exp in instance.__class__.__mro__:
+            return True
+        return isinstance(instance, Pow) and instance.base is S.Exp1
```

Comment:
Are we going to completely replace `exp` with `Pow` eventually? If so, I think we need to raise deprecation warning here.

Reply:
Eventually it will happen. There won't be deprecation warnings right now, because the transition to _Pow(E, x)_ is not yet ready.